rewrite-java-11/src/main/java/org/openrewrite/java/isolated/ReloadableJava11ParserVisitor.java

```diff
@@ -1460,6 +1461,82 @@ public J visitWildcard(WildcardTree node, Space fmt) {
         return new J.Wildcard(randomId(), fmt, Markers.EMPTY, bound, convertOrNull(wildcard.inner));
     }
 
+    /**
+     * --------------
+     * Module Info
+     * --------------
+     */
+    @Override
+    public J visitModule(ModuleTree node, Space space) {
+        skip("module");
+        J.FieldAccess name = convert(node.getName());
+        Space prefix = sourceBefore("{");
+        List<J.Annotation> annotations = convertAll(node.getAnnotations());
+        J.Module.ModuleKind kind = J.Module.ModuleKind.valueOf(node.getModuleType().name());
+        List<JRightPadded<Directive>> directives = new ArrayList<>();
+        if (node.getDirectives() != null) {
+            for (Tree t : node.getDirectives()) {
+                directives.add(convert(t, this::statementDelim));
+            }
+        }
+        Space moduleEnd = sourceBefore("}");
+        return new J.Module(UUID.randomUUID(), prefix, Markers.EMPTY, annotations, kind, name, directives, moduleEnd);
```

Comment:
```suggestion
        return new J.Module(randomId(), prefix, Markers.EMPTY, annotations, kind, name, directives, moduleEnd);
```


`Tree.randomId()` has already been statically imported in this file. Use that